website/content/en/docs/operator-scope.md

```diff
@@ -4,25 +4,19 @@ linkTitle: Operator Scope
 weight: 50
 ---
 
-- [Namespace-scoped operator usage](#namespace-scoped-operator-usage)
-- [Cluster-scoped operator usage](#cluster-scoped-operator-usage)
-  - [Changes required for a cluster-scoped operator](#changes-required-for-a-cluster-scoped-operator)
-  - [Example for cluster-scoped operator](#example-for-cluster-scoped-operator)
-- [CRD scope](#crd-scope)
-  - [CRD cluster-scoped usage](#crd-cluster-scoped-usage)
-  - [Example for changing the CRD scope from namespace to cluster](#example-for-changing-the-crd-scope-from-namespace-to-cluster)
-
 ## Overview
 
 A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator watches and manages resources cluster-wide. Namespace-scoped operators are preferred because of their flexibility. They enable decoupled upgrades, namespace isolation for failures and monitoring, and differing API definitions.
 
 However, there are use cases where a cluster-scoped operator may make sense. For example, the [cert-manager](https://github.com/jetstack/cert-manager) operator is often deployed with cluster-scoped permissions and watches so that it can manage issuing certificates for an entire cluster.
 
+**NOTE**: CustomResourceDefinition (CRD) scope can also be changed to cluster-scoped. See the [CRD scope][crd-scope-doc] document for more details.
+
 ## Namespace-scoped operator usage
 
 This scope is ideal for operator projects which will control resources just in one namespace, which is where the operator is deployed.
 
-> **NOTE:** Initial projects created by `operator-sdk` are namespace-scoped by default which means that it will NOT have a `ClusterRole` defined in the `deploy/role_binding.yaml`.
+**NOTE:** Initial projects created by `operator-sdk` are namespace-scoped by default which means that it will NOT have a `ClusterRole` defined in the `deploy/role_binding.yaml`.
```

Comment:
Even though this line wasn't added here, we should still correct the grammar a bit
```suggestion
**NOTE:** Projects created by `operator-sdk` are namespace-scoped by default which means that they will NOT have a `ClusterRole` defined in `deploy/`.
```

Reply:
not part of the scope of this PR but ok we can address either.